pydictable/field.py

```diff
@@ -332,3 +336,53 @@ def validate(self, field_name: str, value):
         for k, v in value.items():
             self.key_type.validate(None, k)
             self.value_type.validate(None, v)
+
+
+class RegexField(Field):
+    def __init__(self, regex_string: str, *args, **kwargs):
+        super(RegexField, self).__init__(*args, **kwargs)
+        self.regex_string = regex_string
+
+    def from_dict(self, v):
+        return v
+
+    def to_dict(self, v):
+        return v
+
+    def validate_dict(self, field_name: str, v):
+        try:
+            assert re.match(self.regex_string, v), f"{v} for {field_name} should be in proper format"
+        except AssertionError as e:
+            raise AssertionError(str(e))
+
+    def validate(self, field_name: str, v):
+        assert isinstance(v, str)
+
+    def regex(self):
```

Comment:
You can get it from self right?

Reply:
If i get it from self in Field, it will break for other fields, we need that

Reply:
Why it breaks other fields? How is it different from accessing it from self and this function?